pages/common/last.md

```diff
@@ -10,18 +10,18 @@
 
 `last -n {{login_count}}`
 
-- View full login times and dates:
+- Print the full date and time for entries and then display the hostname column last to prevent truncation:
 
-`last -F`
+`last -F -a`
 
-- View the last login by a specific user:
+- View all logins by a specific user and show the ip address instead of the hostname:
 
-`last {{user_name}}`
+`last {{user_name}} -i`
 
-- View the last reboot (last login of the pseudo user reboot):
+- View all recorded reboots (last logins of the pseudo user reboot):
```

Comment:
Maybe we should put the user "reboot" in quotes. Just to clarify that reboot is an user and not the term "user reboot"